e2e/constants/selectors/formBuilder.ts

```diff
@@ -0,0 +1,41 @@
+export const FORMBUILDER_SELECTORS = {
+    // Form editing
+    formTitle: "form-title",
+    formRenameTextField: "form-rename-text-field",
+    formRenameSubmitButton: "form-rename-submit-button",
+    questionHeader: "question-header",
+    
+    // Navigation
+    submissionsTab: "submissions-tab",
+    submissionsCountLabel: "submissions-count-label",
+    moreDropdownIcon: "more-dropdown-icon",
+    analyticsMoreTab: "analytics-more-tab",
+    submissionsInsightsTitle: "submissions-insights-title",
+    
+    // Publishing
+    publishButton: "publish-button",
+    publishPreviewButton: "publish-preview-button",
+    
+    // Single choice options
+    singleChoiceOptionsContainer: "single-choice-options-container",
+    
+    // Multi choice options
+    multiChoiceOptionsContainer: "multi-choice-options-container",
+    
+    //Options
+    contentTextField: "content-text-field",
+    addOptionLink: "add-option-link",
+    addBulkOptionLink: "add-bulk-option-link",
+    bulkAddOptionsTextarea: "bulk-add-options-textarea",
+    bulkAddOptionsDoneButton: "bulk-add-options-done-button",
+    randomizeSwitchLabel: "randomize-switch-label",
+    hideQuestionToggleLabel: "hide-question-toggle-label",
+    
+    // UI elements
+    propertiesHeader: "properties-header",
+    spinner: "neeto-ui-spinner"
+}
+
+export const FORMBUILDER_API = {
+    publishAPI: 'https://neeto-form-web-playwright.neetodeployapp.com/api/v1/forms/'
+}
```

Comment:
What is the need for APIs?

Even if it is required, this should come under constants/routes.ts. I had already mentioned it in the video I made earlier.

Also, all URLs should be starting with a `/` if it is a part of the application since we have already declared a base URL.

```suggestion
export const FORMBUILDER_API = {
    publishAPI: '/api/v1/forms/'
}
```